### PR TITLE
Bucketlist api

### PR DESCRIPTION
**Pull request by Sharonsyra**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        # Have to re-enable the standard pragma
+        pragma: no cover
+
+        if 0:
+        if __name__ == .__main__.:
+        if __name__==.__main__.:
+
+
+
+ignore_errors = True
```
Comment (bmwenda) [inline .coveragerc:23]: Add a new line after line 22. Do this for all other recurring cases.

File: app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+            else:
+                message = user_id
+                response = {
+                    'message': message
+                }
+                return make_response(jsonify(response)), 401
+
+    @app.route('/api/v1.0/bucketlists/<int:id>/items/', methods=['POST', 'GET'])
+    def items(id, **kwargs):
+        auth_header = request.headers.get('Authorization')
```
Comment (bmwenda) [inline app/__init__.py]: Is there need for the **kwargs?

File: app/models.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            }
+            jwt_string = jwt.encode(
+                payload,
+                current_app.config.get('SECRET'),
+                algorithm='HS256'
+            )
+            return jwt_string
+
+        except Exception as e:
+            return str(e)
```
Comment (bmwenda) [inline app/models.py:51]: Be more specific in the exception you are raising.

File: app/__init__.py (hunk 2 above)
Comment (Sharonsyra) [reply app/__init__.py]: fixing that thanks!

File: app/models.py (hunk 3 above)
Comment (Sharonsyra) [reply app/models.py:51]: Working on that

**Pull request merged**